src/roll20/content-script.js

```diff
@@ -248,17 +248,21 @@ function rollSkill(request, custom_roll_dice = "") {
             prof_val += "+[[" + request.character.proficiency + " * 2]]";
             reliableTalent = request.reliableTalent;
         }
-        const d20 = reliableTalent ? "{1d20, 0d0 + 10}kh1" : "1d20";
+        const d20_roll = reliableTalent ? "{1d20, 0d0 + 10}kh1" : "1d20";
+        if (request.character["racial-traits"].includes("Lucky"))
+            d20_roll = d20_roll.replace(/1d20+/g, "$&r1");
```

Comment:
not using the same regexp as above, but also you added to it `ro<=1` above (in roll renderer) and now you add `r1`.. I assume it's because `r1` is the specific roll20 syntax? the regexp wouldn't need to be different though, and `1d20+` means it would match `1d200000000` so the regexp is wrong